apply/processor/gen.go

```diff
@@ -88,10 +92,66 @@ func (g *GenerateProcessor) GetPipeLine() ([]func(cluster *v2.Cluster) error, er
 		g.MountRootfs,
 		g.ApplyRegistry,
 		g.UnmountImage,
+		g.CmdAddRegistryHosts,
+		g.CopyK8sConfFileToDefultCluster,
 	)
 	return todoList, nil
 }
 
+func (g *GenerateProcessor) CopyK8sConfFileToDefultCluster(cluster *v2.Cluster) error {
+	eg, _ := errgroup.WithContext(context.Background())
+	masters := cluster.GetMasterIPList()
+	for _, master := range masters {
+		master := master
+		eg.Go(func() error {
+			ssh, err := ssh.GetHostSSHClient(master, cluster)
+			if err != nil {
+				return fmt.Errorf("new ssh client failed %v", err)
+			}
+			KubeAdminConf := fmt.Sprintf(common.RemoteCmdCopyStatic, filepath.Join(common.KubeAdminConf), filepath.Join(common.DefaultClusterBaseDir(cluster.Name)))
```

Comment:
The same `kubeAdminConf`, `KubeControllerConf` are created each time through the loop. . .
Create these same fixed values outside the loop.
optimize writing.

ex.
```
copyCmds := []string{}
for _, f := []string{common.KubeControllerConf, common.KubeScheduleConf...} {
copyCmd := fmt.Sprintf(common.RemoteCmdCopyStatic, f, common.DefaultClusterBaseDir(cluster.Name))
copyCMds = append(copyCmds, copyCmd)
}
```